lib/src/components/NxTag/NxTag.scss

```diff
@@ -64,6 +64,10 @@
 
 .nx-tag__input {
   @include nx-hide-checkbox-helper.hide-checkbox;
+  // This fixed the checkbox shifting the truncated text
+  // to the left when focused on Chrome.
+  position: absolute;
+  right: 0;
```

Comment:
Weird, I wonder if if this a regression in more recent versions of Chrome.  I doubt we wouldn't have noticed this issue when we first created `NxTag`.  IMO we should have a visual test that would have caught this

Reply:
> Weird, I wonder if if this a regression in more recent versions of Chrome. I doubt we wouldn't have noticed this issue when we first created `NxTag`. IMO we should have a visual test that would have caught this

Added an extra visual test  here: f907f61fec9666bba6ca301b7c1036a8c000096c